Decisions.Equifax/ConsumerCreditReport/ConsumerCreditReportSteps.cs

```diff
@@ -26,137 +16,10 @@ public class ConsumerCreditReportSteps
         /// </summary>
         public static ConsumerCreditReportResponse GetConsumerCreditReport(ConsumerCreditReportRequest request)
         {
-            return ExecuteCreditReportRequest(request);
-        }
-
-
-
-        /// <summary>
-        /// Helper: Execute API Endpoint
-        /// </summary>
-        private static ConsumerCreditReportResponse ExecuteCreditReportRequest(ConsumerCreditReportRequest request)
-        {
-            JsonSerializerSettings jsonSettings = new JsonSerializerSettings
-            {
-                NullValueHandling = NullValueHandling.Ignore
-            };
-            
             string scope = ModuleSettingsAccessor<EquifaxSettings>.Instance.EquifaxConsumerCreditReportScope;
```

Comment:
We should check EquifaxConsumerCreditReportScope and EquifaxConsumerCreditReportEndpoint to make sure it has a value here. Even with the validation on the Settings object, we may need to throw an exception to remind them to go set that up before trying to use the step.

We should do this in both steps.